src/locales/en_GB/messages.json

```diff
@@ -112,6 +112,12 @@
     "message": "Copy value",
     "description": "Copy value to clipboard"
   },
+  "minimizeOnCopyToClipboard": {
```

Comment:
revert this file. Only need to make changes to the `en` locale.

Reply:
Okay.